R/clear.cache.R

```diff
@@ -65,3 +65,8 @@ clear.cache <- function (...){
         }
 }
 
+# Determine if a variable is cached or not
+.is.cached <- function (variable) {
+        file.exists(file.path(.cache.dir, paste0(variable, ".RData")))
```

Comment:
Good spot @Hugovdberg.  You're quite right.  

When I was coding the caching I had an `.is.cached` function, but it became apparent that you need to know a lot more than the presence of the file (e.g. is it stale or not).  So `.is.cached` morphed into `read.cache.info`.

I'll call the function directly even though it does more work than I need here.  It's simpler than creating a new function and refactoring.  The code would read something like `if (.read.cache.info(var)$in.cache) ...` which is reasonably readable.